Update assist.py

Bram-diederik opened the pull request:
added ssl support
fixed direct command handling

Bram-diederik committed:
Update assist.py

added ssl support
fixed direct command handling

Commit:
added a service and a nice reload with out loging out option.

Commit:
added agent selection

Commit:
--cli and --new options for command line interaction.
needed for my asterisk integration

Bram-diederik committed:
Update README.MD

Commit:
added --env

Commit:
Merge remote-tracking branch 'refs/remotes/origin/main'

Bram-diederik committed:
Update README.MD

old forked assist_cmd.py that does not exist.

thomas-mardis commented:
Hey nice job!! So sorry for the slow response on this. I just saw this in my inbox, annoyed at myself for not catching this. I'll look it over and merge it in!

falldeaf commented:
Hey so sorry it took so long to response to your PR. These look like great updates! I'll review it asap!

Bram-diederik commented:
I did all coding with ai. There might be some regression. The screenshot you made is off.

Bram-diederik commented:
Note. I made some changes to allow a better assist experience over voip with asterisk. (Voip system)
https://github.com/Bram-diederik/home-assistant-asterisk-assist

If you can figure out a way to send an area id im not stopping you.

Commit:
oops. agent did not remember conversation id when in interactive mode

Commit:
added a session id for multi session usage

Commit:
fixed the ignoring of the default agent in the enviorment file

Commit:
-ci typo

Commit:
oops

Commit:
oops again

Commit:
Oke I hijack this repo to make a full dashboard.
assist.py code is unchanged so can still be used.

Commit:
env example

Commit:
old place

Bram-diederik committed:
Revise README with new project name and info

Updated project name and added documentation link.

Commit:
I listen to feedback
I added a F4 to copy info to a txt log.
also added a Help settings and init setup new F1
F2 is now the assist
F3 ha commander

Commit:
install for debian
